.github/workflows/android-perf.yml

```diff
@@ -268,6 +278,6 @@ jobs:
       # TODO: Hard code llm_demo_bpe for now in this job.
       android-app-archive: https://gha-artifacts.s3.amazonaws.com/${{ github.repository }}/${{ github.run_id }}/artifact/llm_demo_bpe/app-debug.apk
       android-test-archive: https://gha-artifacts.s3.amazonaws.com/${{ github.repository }}/${{ github.run_id }}/artifact/llm_demo_bpe/app-debug-androidTest.apk
-      test-spec: ${{ inputs.test_spec }}
+      test-spec: ${test_spec}
```

Comment:
This syntax looks wrong as GitHub uses something like ${{ env.test_spec }} in their yaml workflow instead (https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/store-information-in-variables).  I think the fix could be to move the default value here, i.e. https://github.com/pytorch/executorch/pull/4736

Reply:
Aha, I don't get a chance to really test it yet. If you already got a proper fix, let's go for it.